### PR TITLE
Solved PS520

### PR DESCRIPTION
**Pull request by kjawadDeveloper2**

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -53,12 +53,12 @@ android {
     }
 
     signingConfigs {
-        debug  {
-            storeFile file('keystore.jks')
-            storePassword 'tech.pylons'
-            keyAlias 'key_tech_pylon'
-            keyPassword 'tech.pylons'
-        }
+//        debug  {
```
Comment (mustafapylons) [inline android/app/build.gradle]: Is the debug code necessary, or is it dead code?

File: lib/screens/creator_hub/widgets/draft_list_tile.dart [hunk 2]
```diff
@@ -63,49 +70,47 @@ class DraftListTile extends StatelessWidget {
...
-                      backgroundColor: EaselAppTheme.kWhite,
-                      ar: false,
-                      autoRotate: false,
-                      cameraControls: false,
-                    ),
-                    assetType: nft.assetType.toAssetTypeEnum(),
-                  )
-                ),
+                      onVideo: (context) => CachedNetworkImage(
+                        fit: BoxFit.fill,
```
Comment (mustafapylons) [inline lib/screens/creator_hub/widgets/draft_list_tile.dart]: It would be better to write more reusable code here for multiple conditions.

**Review by mustafapylons — COMMENTED** [review]
Few things to fix, otherwise code is good.

File: android/app/build.gradle (hunk 1 above)
Comment (Amean) [inline android/app/build.gradle]: Should this dead code be here?

Comment (kjawadDeveloper2) [reply android/app/build.gradle]: We un comment this when we need to release the build.

Comment (kjawadDeveloper2) [reply android/app/build.gradle]: We un comment this when we need to release the build.  This makes the build a little bit faster during testing.

**Review by Amean — APPROVED** [review]

**Review by mustafapylons — APPROVED** [review]
Looks good to me now.

**Comment by kjawadDeveloper2** [comment]
Some design issues i saw while running the app.

**Pull request merged**